subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -75,8 +75,11 @@ static int download_client_callback(const struct download_client_evt *event)
 		if (err != 0) {
 			LOG_ERR("dfu_target_write error %d", err);
 			err = download_client_disconnect(&dlc);
+			if (err != 0) {
+				LOG_ERR("Unable to disconnect download_client");
```

Comment:
I agree with @lemrey  that the logging done in download client is enough, so remove the check here, and don't assign anything to the return value of the disconnect. Do we need to explicitly ignore the return value, or is it acceptable to simply not assign the return value to a variable?

Reply:
Not assigning is fine. `(void)` is good form.